src/maven_tests/testmzkitchenprocessor/testmzkitchenprocessor.cpp

```diff
@@ -0,0 +1,58 @@
+#include <fstream>
+#include <vector>
+#include <map>
+#include <ctime>
+#include <limits.h>
+#include <sstream>
+
+#include "mzSample.h"
+#include "parallelMassSlicer.h"
+#include "../maven/projectDB.h"
+
+#include "../maven/classifierNeuralNet.h"
+#include "../maven/database.h"
+#include "./Fragment.h"
+#include "pugixml.hpp"
+#include <sys/time.h>
+
+#include <QtCore>
+#include <QCoreApplication>
+#include <QPixmap>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+//fields
+Database DB;
+static string mzkitchenMspFile = "";
+
+//functions
+void processCLIArguments(int argc, char* argv[]);
+void mzkitchenSearch();
+
+int main(int argc, char* argv[]) {
+    processCLIArguments(argc, argv);
+    mzkitchenSearch();
+    cout << "Test Passed" << endl;
```

Comment:
I didn't see any assertion for test, does it mean, we will always see "Test passed" message?

Reply:
@feicalico Yeah... I never got around to adding any `assert()` statements. So, this test isn't very informative right now :-)

Reply:
The idea would be if I had added assert statements, then you would reach the `Test Passed` string. if `mzkitchenSearch()` causes the program to crash, then the `Test Passed` string will never be executed. So, currently, this test is only verifying that the `mzkitchenSearch()` method does not cause the program to crash.

Reply:
hmm,  looks like "mzkitchenSearch" is a test helper function, what's the exact function you want to test?  DB.loadNISTLibrary?

Reply:
Yes. Previously, I built a big testing framework in cmake, but I couldn't test Qt associated components, which Db.loadNISTLibrary()  requires. There is additional testing here:
https://github.com/calico/mass_spec/blob/master/lib/cpp/mzkit/libmzkit/lc_test_mzkitchenScoring.cpp

Reply:
I considered porting all of the tests into the `maven_tests` subproject, but this would have taken a long time, and I didn't know if it is worth doing (at least right now). For new features/functionality, I'll add tests to `maven_tests` instead of the `mass_spec` testing harness.